packages/code-du-travail-frontend/src/common/ServiceRenseignement.js

```diff
@@ -61,28 +65,30 @@ function ServiceRenseignement() {
           Service gratuit <span>+ prix appel</span>
         </Pricing>
       </NumberInsert>
-
       <Heading as="h4">Contact par email et prise de rendez-vous</Heading>
-      <p>
-        <UID name={(id) => `id_${id}`}>
-          {(id) => (
-            <>
-              <label htmlFor={id}>
-                Saisissez votre numéro de département&nbsp;:
-              </label>
-              <StyledInput
-                id={id}
-                type="text"
-                name="search-service"
-                autoComplete="off"
-                maxLength="3"
-                placeholder="ex. 31, 35, 75"
-                onChange={onDepartmentInput}
-              />
-            </>
-          )}
-        </UID>
-      </p>
+      <Form
```

Comment:
There 2 form tag, only one is needed

Reply:
It's from the sample of react-final-forms `https://final-form.org/docs/react-final-form/getting-started`.  I miss something ? Perhaps you mean I don't need react-final-form here ?

Reply:
Oops sorry It's been a long time since I work with final form -_-'